FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,39 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case .success((let data, let response)):
+				if response.statusCode == 200,
+				   let root = try? JSONDecoder().decode(Root.self, from: data) {
+					completion(.success(root.feedImages))
+				} else {
+					completion(.failure(RemoteFeedLoader.Error.invalidData))
```

Comment:
The `RemoteFeedLoader` type can be removed since `Error` is defined under the `RemoteFeedLoader` namespace:

```suggestion
					completion(.failure(Error.invalidData))
```